l10n.toml

```diff
@@ -163,7 +163,7 @@ branches = [
 
 [[paths]]
   reference = "components/feature/addons/src/main/res/values/strings.xml"
-  l10n = "components/feature/addons/src/main/res/values/values-{android_locale}/strings.xml"
+  l10n = "components/feature/addons/src/main/res/values-{android_locale}/strings.xml"
```

Comment:
@Pike I noticed that I put the wrong path on the `l10n.toml` file. I'm moving the files to the right place on this pr, I believe you have to do the same on your end sorry for that.

Reply:
So this is gonna be an interesting experience, 'cause you're the first one trying.

CC @mathjazz to make sure he's aware of this coming in. Also @flodolo.

We might need to remove files by hand.

On a positive note, the values-foo files are only written, not read, so we won't loose any data, AFAICT.

Reply:
Without [work on Pontoon's side](https://mozilla-l10n.github.io/documentation/tools/pontoon/renaming_a_file.html), are we going to lose a bunch of things? History, attributions, pending suggestions.

Reply:
The Resource path doesn't change, just where Pontoon syncs l10n files to, and reads from.

This would be a problem if we had changes coming in to Pontoon through VCS, but for android-l10n we don't, so no problem.

I think.

Famous last words.

Related, that downloads on projects with project config are broken, https://bugzilla.mozilla.org/show_bug.cgi?id=1530988.

Reply:
Is OK to land as it is?

Reply:
Yes, thanks.

I started feeling bad for you for all that jazz flying by, none of which is actionable for you.

Reply:
I also believe this should JustWork™. We might want to test on stage first, though. TM

Reply:
No worries!